src/utils/hooks/useListApplications.tsx

```diff
@@ -6,13 +6,19 @@ import {
 } from '../../utils/generated/graphql'
 import { ApplicationDetails } from '../types'
 
-const useListApplication = () => {
+interface UseListApplicationsProps {
+  type?: string
+}
+
+const useListApplication = ({ type }: UseListApplicationsProps) => {
   const [applications, setApplications] = useState<ApplicationDetails[] | undefined>()
   const [loading, setLoading] = useState(true)
   const [error, setError] = useState('')
 
   const { data, loading: apolloLoading, error: apolloError } = useGetApplicationsQuery({
+    variables: { code: type as string },
     fetchPolicy: 'network-only',
+    skip: !type,
```

Comment:
I assume we have an issue to set a default type based on user permissions, right? (So we won't need to skip)

Reply:
We don't have a specific one for that. We just have the #251 that would deal with that, that's what I was thinking. If you feel we should have a issue just to deal with this problem, let me know.